website/vercel.json

```diff
@@ -15,6 +15,17 @@
     }
   ],
   "redirects": [
+    {
+      "source": "/",
+      "has": [
+        {
+          "type": "host",
+          "value": "www.featureflagbook.com"
+        }
+      ],
+      "destination": "https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices",
+      "permanent": true
```

Comment:
```suggestion
      "permanent": false
```

Reply:
@alvinometric changing this to non-permanent in case the host matcher does not work how I hope it will.